src/components/CodeSuccessModal/CodeSuccessModal.jsx

```diff
@@ -0,0 +1,15 @@
+import balloons from '../../images/balloons.png';
+import styles from './CodeSuccessModal.module.scss'
+
+
+export default function CodeSuccessModal({ onConfirm }) {
```

Comment:
can we reuse existing `ModalWindow` component ? 

by design OTP modal looks the same as the existing ModalWindow:
- same grey overlay 
- same close button
- same main white pane

![image](https://github.com/user-attachments/assets/5d16fb7c-c493-438f-9f09-0ec55dd2f6f2)